sellotape/sellotape_dj/main_app/models.py

```diff
@@ -1,39 +1,49 @@
 from django.db import models
 from django.contrib.auth.models import User
 
-# https://docs.djangoproject.com/en/dev/topics/auth/customizing/#extending-the-existing-user-model
+
 class Profile(models.Model):
     # Inherits Attributes from User default instance
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     # User avatar, optional: limit the image instance height and width
     avatar = models.ImageField()
     country = models.CharField(max_length=100, default='Israel')
-    youtube_link = models.URLField(max_length=200)
-    twitch_link = models.URLField(max_length=200)
-    
+    youtube_link = models.URLField(max_length=200, blank=True)
+    twitch_link = models.URLField(max_length=200, blank=True)
+
     class City(models.IntegerChoices):
         tel_aviv = '1', 'Tel Aviv'
         hamerkaz = '2', 'HaMerkaz'
         hadarom = '3', 'HaDarom'
         hatzafon = '4', 'HaTzafon'
-    
+
     city = models.IntegerField(
         choices=City.choices,
         default=City.tel_aviv,
-    )   
+    )
+
 
 class UserFollowers(models.Model):
-    user_followers_id = models.AutoField(primary_key=True)
-    user_id = models.ForeignKey('self', on_delete=models.CASCADE)
-    follower_id	= models.ForeignKey(Profile, on_delete=models.CASCADE)
+    follow_from = models.ForeignKey(Profile, related_name='follow_from', on_delete=models.CASCADE)
+    follow_to = models.ForeignKey(Profile, related_name='follow_to', on_delete=models.CASCADE)
```

Comment:
I think this is a bit confusing.. how about a simple `user` and `follows` fields?

Reply:
changed to `user_id` and `follows`.